### ISSUE TITLE
While focussing on password filed on webpage and open Maccy, its blink and disappear  

### ISSUE THREAD

**Issue by deven-softices** [p0deje/Maccy#864]
### Before Submitting Your Bug Report

- [X] I have verified that there isn't already an issue reporting the same bug to prevent duplication.
- [X] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Maccy Version (see 'About' window)

1.0.0 (31)

### macOS Version

14.6.1 (23G93)

### Maccy Settings

```Shell
{
    "KeyboardShortcuts_delete" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":51}";
    "KeyboardShortcuts_pin" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":35}";
    "KeyboardShortcuts_popup" = "{\\"carbonKeyCode\\":9,\\"carbonModifiers\\":768}";
    "LaunchAtLogin__hasMigrated" = 1;
    "NSStatusItem Preferred Position Item-0" = 577;
    "NSWindow Frame com.sindresorhus.Settings.FrameAutosaveName" = "1048 382 542 437 0 0 2048 1255 ";
    SUEnableAutomaticChecks = 1;
    SUHasLaunchedBefore = 1;
    SULastCheckTime = "2024-08-30 08:50:01 +0000";
    SUUpdateGroupIdentifier = 4235031747;
    enabledPasteboardTypes =     (
        "public.html",
        "public.rtf",
        "public.utf8-plain-text",
        "public.tiff",
        "public.file-url",
        "public.png"
    );
    ignoredPasteboardTypes =     (
        "com.typeit4me.clipping",
        "net.antelle.keeweb",
        "Pasteboard generator type",
        "com.agilebits.onepassword",
        "de.petermaurer.TransientPasteboardType"
    );
    migrations =     {
        "2020-04-25-allow-custom-ignored-types" = 1;
        "2020-06-19-use-keyboardshortcuts" = 1;
        "2020-09-01-ignore-keeweb" = 1;
        "2021-02-20-allow-to-customize-supported-types" = 1;
        "2021-06-28-add-title-to-history-item" = 1;
        "2021-10-16-remove-dynamic-pasteboard-types" = 1;
        "2022-08-01-rename-suppress-clear-alert" = 1;
        "2022-11-14-add-html-rtf-to-supported-types" = 1;
        "2023-01-22-add-regexp-search-mode" = 1;
    };
    pasteByDefault = 1;
    removeFormattingByDefault = 1;
    searchMode = fuzzy;
}
```

### Description

on any of the webpage if i open Maccy for past text it works, but while keyboard cursor is on password field (password field focus) and if i open Maccy by keyboard shortcut or via app icon it blink and disapprear as many time i open .

### Steps to Reproduce

1. open any website with login form
2. focus on password field 
3. open Maccy via keyboard shortcut or via app icon

**Comment by p0deje** [comment]
This will be fixed in 2.0, you can try upgrading to the latest beta https://github.com/p0deje/Maccy/releases/download/2.0.0.beta.13/Maccy.app.zip

**Comment by deven-softices** [comment]
Thanks for the update, its working fine with beta version
